scripts/text_prompt_walk_to_object.py

```diff
@@ -0,0 +1,266 @@
+from bosdyn.client.manipulation_api_client import ManipulationApiClient
+from bosdyn.client.robot_command import (RobotCommandBuilder, RobotCommandClient,
+                                         block_until_arm_arrives, blocking_stand)
+
+from conq.clients import Clients
+from conq.hand_motion import hand_pose_cmd
+from conq.manipulation import blocking_arm_command
+from conq.cameras_utils import get_color_img, RGB_SOURCES, DEPTH_SOURCES
+
+
+g_image_click, g_image_display = None, None
+
+
+def walk_to_object(config):
+    """Get an image and command the robot to walk up to a selected object.
+       We'll walk "up to" the object, not on top of it.  The idea is that you
+       want to interact or manipulate the object."""
+
+    # See hello_spot.py for an explanation of these lines.
+    bosdyn.client.util.setup_logging(config.verbose)
+
+    sdk = bosdyn.client.create_standard_sdk('WalkToObjectClient')
+    robot = sdk.create_robot(config.hostname)
+    bosdyn.client.util.authenticate(robot)
+    robot.time_sync.wait_for_sync()
+
+    assert robot.has_arm(), 'Robot requires an arm to run this example.'
+
+    # Verify the robot is not estopped and that an external application has registered and holds
+    # an estop endpoint.
+    assert not robot.is_estopped(), 'Robot is estopped. Please use an external E-Stop client, ' \
+                                    'such as the estop SDK example, to configure E-Stop.'
+
+    lease_client = robot.ensure_client(bosdyn.client.lease.LeaseClient.default_service_name)
+    image_client = robot.ensure_client(ImageClient.default_service_name)
+    manipulation_api_client = robot.ensure_client(ManipulationApiClient.default_service_name)
+
+    with bosdyn.client.lease.LeaseKeepAlive(lease_client, must_acquire=True, return_at_exit=True):
+        # Now, we are ready to power on the robot. This call will block until the power
+        # is on. Commands would fail if this did not happen. We can also check that the robot is
+        # powered at any point.
+
+        robot.logger.info('Powering on robot... This may take a several seconds.')
+        robot.power_on(timeout_sec=20)
+        assert robot.is_powered_on(), 'Robot power on failed.'
+        robot.logger.info('Robot powered on.')
+
+        # Tell the robot to stand up. The command service is used to issue commands to a robot.
+        # The set of valid commands for a robot depends on hardware configuration. See
+        # RobotCommandBuilder for more detailed examples on command building. The robot
+        # command service requires timesync between the robot and the client.
+
+        robot.logger.info('Commanding robot to stand...')
+        command_client = robot.ensure_client(RobotCommandClient.default_service_name)
+        blocking_stand(command_client, timeout_sec=10)
+        robot.logger.info('Robot standing.')
+
+        status = open_gripper(Clients) #FIXME #BUG: Open the gripper before capturing image
```

Comment:
Does this work? this is the class name `Clients` not an instance of that class

Reply:
Nope, it's not working. Instead of just opening the gripper here, I plan to define a gaze pose to look at the subject and then open the gripper in the same action sequence. Will replace this with that entire thing later.